src/test/java/com/ashkan/moviesapi/MoviesAPITests.java

```diff
@@ -4,7 +4,7 @@
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
-class MoviesapiApplicationTests {
+class MoviesAPITests {
```

Comment:
Should be good to have a couple of unit tests for Service and Controller layers.